AdventOfCode2023/Day7.cs

```diff
@@ -0,0 +1,176 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace AdventOfCode2023
+{
+    internal class Day7
+    {
+        public static long Day7Main(string[] inputArray)
+        {
+            var hands = new List<Hand>();
+            long totalSum = 0;
+
+            foreach (var input in inputArray)
```

Comment:
should read up on linq, it's super powerful and you can covert this foreach loop to something like 

var hands = inputArray.Select(input => 
    new Hand(input.Split(' ')[0], Convert.ToInt32(input.Split(' ')[1]))
).ToList();